pkg/sink/sink.go

```diff
@@ -235,10 +234,12 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 
 	} else {
 		responseEvent := cloudevents.NewEvent()
-		responseEvent.SetID(uuid.New().String())
-		responseEvent.SetSource(r.EventListenerName)
+		responseEvent.SetID(elUID)
+		responseEvent.SetSubject(r.EventListenerNamespace + "." + r.EventListenerName + " processing " + elUID)
```

Comment:
IIRC subject is not mandatory.
Required are:
spec-version (set by "constructor" automatically)
ce-type
ce-source
ce-id

Reply:
ce-subject and ce-time are optional, but it might be nice to have them